library/core/src/iter/adapters/step_by.rs

```diff
@@ -32,16 +42,154 @@ where
 
     #[inline]
     fn next(&mut self) -> Option<Self::Item> {
+        self.spec_next()
+    }
+
+    #[inline]
+    fn size_hint(&self) -> (usize, Option<usize>) {
+        self.spec_size_hint()
+    }
+
+    #[inline]
+    fn nth(&mut self, n: usize) -> Option<Self::Item> {
+        self.spec_nth(n)
+    }
+
+    fn try_fold<Acc, F, R>(&mut self, acc: Acc, f: F) -> R
```

Comment:
Should this also have inline?
```suggestion
    #[inline]
    fn try_fold<Acc, F, R>(&mut self, acc: Acc, f: F) -> R
```

Reply:
This one didn't have it before. But I'll add it to the concrete impls.